Makefile

```diff
@@ -1,33 +1,39 @@
 include common.mk
 
+STAGE=${DSS_DEPLOYMENT_STAGE}
```

Comment:
Nit: Could probably just set `DSS_DEPLOYMENT_STAGE` directly and get rid of `STAGE`.